src/libasr/codegen/wasm_to_x86.cpp

```diff
@@ -68,6 +69,51 @@ class X86Visitor : public WASMDecoder<X86Visitor>,
         }
     }
 
+    void visit_I32Eqz() {}
+
+    void visit_Unreachable() {}
+
+    void visit_EmtpyBlockType() {}
+
+    bool is_else_stmt_available() {
+        auto offset_ = offset;
+        uint8_t cur_byte = wasm::read_b8(code, offset_);
+        while (cur_byte != 0x0B) {
+            if (cur_byte == 0x05) {
+                cur_byte = wasm::read_b8(code, offset_);
+                if (cur_byte == 0x0B || cur_byte == 0x41) {
+                    return false;
+                } else {
+                    return true;
+                }
+            }
+            cur_byte = wasm::read_b8(code, offset_);
+        }
```

Comment:
But this logic, detects the else block, even for the following syntax!!
```
if True:
	pass
```

Reply:
That is the reason, I check the next byte to be 0x0B (which checks the end of the if stmt).

Reply:
At present, if there is no `else` block, then `visit_Else()` should not be visited. I guess, we might be missing something.